user-guide/content-curation/external-collection-indexing/examples/esi-mapserver-config.json

```diff
@@ -0,0 +1,68 @@
+{
+  "type": "Collection",
+  "id": "global_esi_4wk",
+  "stac_version": "1.1.0",
+  "description": "The Evaporative Stress Index (ESI) reveals regions of drought where vegetation is stressed due to lack of water. The ESI can capture early signals of flash drought and is produced weekly at 5-kilometer resolution for the entire globe.",
+  "stac_extensions": [
+    "https://stac-extensions.github.io/render/v2.0.0/schema.json"
+  ],
+  "links": [
+    {
+      "rel": "wms",
+      "href": "https://gis1.servirglobal.net/arcgis/services/Global/ESI_4WK/MapServer/WMSServer?request=GetCapabilities&service=WMS",
+      "type": "image/png",
+      "title": "Visualized through a WMS",
+      "wms:layers": [
+        "1",
+        "2",
+        "3"
+      ],
+      "wms:styles": [
+        "default"
+      ]
+    },
+    {
+      "rel": "via",
+      "href": "https://gis1.servirglobal.net/arcgis/rest/services/Global/ESI_4WK/MapServer",
+      "type": "text/html",
+      "title": "Parent ArcGIS server url"
+    }
+  ],
+  "renders": {
+    "image": {
+      "layers": "1"
+    },
+    "footprint": {
+      "layers": "2"
+    },
+    "boundary": {
+      "layers": "3"
+    }
+  },
```

Comment:
```suggestion
```

Are we doing this in any collections now? The extension only covers tiler parameters today https://github.com/stac-extensions/render and I think the collection this creates would be invalid

Reply:
I'm still trying to catch up on what was implemented in pyarc2stac but this layers metadata is coming from pyarc2stac https://github.com/NASA-IMPACT/pyarc2stac/blob/b56080192b840191c13ed7e599aa566ec9e29ac7/pyarc2stac/ArcReader.py#L207-L223

Sorry to drag out this review. I think we now need to decide if we are going to propose changes to the renders extension or not. Also I will try to determine whether or not the layers-in-renders metadata are used in the ui code